### DISCUSSION TITLE
UI scale differences between Windows and Skia+Gtk head

### DISCUSSION THREAD

**Discussion by chrisgregan** [unoplatform/uno#13932] [Q&A] [answered]
Hi, thanks for making Uno Platform - it's awesome!

I've made a test solution using the latest Uno Templates using Visual Studio 7.6.4 and Uno Platform 4.10.10.11. I've succeeded in building and running the Windows and Skia.Gtk heads on Windows 11. 

The UI scale is very different for Windows and Skia.Gtk:
![image](https://github.com/unoplatform/uno/assets/3728355/0400743c-f448-4435-852e-8ad21c60fdb8)

The Skia.Gtk window appear very small, in fact it's barely usable. I am using a 4k monitor with the Windows scale setting set to 175%
![image](https://github.com/unoplatform/uno/assets/3728355/dbbfa253-877a-42f1-976f-cb0625792e38)

Does anyone know why this is happening and is there a fix for it?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. This is tracked in https://github.com/unoplatform/uno/issues/9218.
